### PR TITLE
LocUpdFgService Updated

### PR DESCRIPTION
**Pull request by fullmetaltac**
LocUpdFgService Updated

### CODE REVIEW

File: android-o/AndroidPlayLocation/LocUpdFgService/LocUpdFgService/LocationUpdatesService.cs [hunk 1]
```diff
@@ -109,12 +114,20 @@ public override void OnCreate()
...
 			HandlerThread handlerThread = new HandlerThread(Tag);
 			handlerThread.Start();
 			ServiceHandler = new Handler(handlerThread.Looper);
-			NotificationManager = (NotificationManager) GetSystemService(NotificationService);
-		}
+			NotificationManager = (NotificationManager) GetSystemService(NotificationService);
+
+		    if (Build.VERSION.SdkInt >= Build.VERSION_CODES.O)
+		    {
+		        var name = (String)GetString(Resource.String.app_name);
```
Comment (pjcollins) [inline android-o/AndroidPlayLocation/LocUpdFgService/LocUpdFgService/LocationUpdatesService.cs]: This cast is throwing on startup, we'll need to update it.

Comment (fullmetaltac) [reply android-o/AndroidPlayLocation/LocUpdFgService/LocUpdFgService/LocationUpdatesService.cs]: Done.

**Comment by fullmetaltac** [comment]
I fixed problem with runtime error on my device.
image

**Comment by fullmetaltac** [comment]
I was receiving error that described on stackoverflow:
https://stackoverflow.com/questions/41163887/could-not-resolve-type-with-token-0100000f
So I cleaned solution and restored nuget packages.
Changed target and minimum version for my phone.
After that service was working correctly.

**Comment by pjcollins** [comment]
Replaced by #252

**Pull request closed without merging**